system/conf/database.conf

```diff
@@ -2,9 +2,9 @@
 // Configuration file
 //----------------------------------------------------------------------------
 
-host     : 127.0.0.1
+host     : melia-database
 user     : root
-pass     : 
+pass     : 123456
```

Comment:
Same as with the PHP options, you don't need to modify the system configs if you just create the respective user conf files.